build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -56,6 +62,8 @@
                             SetupRestoreTestsInputs;"
           Inputs="@(RestoreTestsInputs)"
           Outputs="@(RestoreTestsInputs->'%(RelativeDir)project.lock.json')">
+    <CallTarget Targets="CleanTestProjectsLockFiles"/>
```

Comment:
Will this incur into the cache targets condition for msbuild? I mean, if this gets called in multiple places like this, will it only run once?